### ISSUE TITLE
Provide a flatpak file

### ISSUE THREAD

**Issue by WPFilmmaker** [ccbogel/QualCoder#6]
Maybe at this state is still too early but I would recommend to eventually release QualCoder as flatpak so that with one click users can install it on their system regarding of the type of distros.

The instructions to build are not hard to follow, but they are definitively not something the average user deals with. Furthermore it require downloading dependencies which means more terminal use. 

Many apps released as flatpak they also have a page on https://flathub.org/ (again more promotion for your project).

**Comment by ccbogel** [comment]
Yes I might give flatpak a trial shortly.

**Comment by WPFilmmaker** [comment]
Good! If you need testing for the flatpak on lubuntu let me know.

**Comment by ccbogel** [comment]
I have had some trouble developing a flatpak. I have created a debian package installer. The file is called QualCoder-1.0.deb, so this should install via a double-click or through your package manager. When you are able, let me know if this installs without errors (appears to be fine on my Mint system).

**Comment by WPFilmmaker** [comment]
I will let you know asap, it seems that my virtualbox error can be fixed by switching to an older kernel version. I saw you uploaded the .deb! I think it makes sense to specify in the "installation" sections that those who use ubuntu or debian can use the deb file above. It is easy to overlook such greyish text :D

**Comment by ccbogel** [comment]
OK I will close this issue as a flatpak is in the distant future. I am doing a lot of reading about how to prepare a debian package that is accepted into the debian official repository, and it seems like quite a bit of work. At least the qualcoder-1.1.deb seems to work ok on my machines including Lubuntu in the virtualbox.

**Comment by spipau** [comment]
Flatpak became the standard to install software under Linux, would be great to see this soon happening. Thank you very much for your amazing work :handshake: :heart:
